examples/mixture_models/dirichlet_mixture_of_multinomials.ipynb

```diff
@@ -0,0 +1,1815 @@
+{
```

Comment:
<p>Just adding some comments to help readers:</p><pre class="ql-syntax" spellcheck="false">n_samples = 10 # nbr of forests observed
total_count = 50 # nbr of trees observed in each forest
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>